extras/lambda-buffers-utils/src/LambdaBuffers/Utils/Logger.hs

```diff
@@ -0,0 +1,41 @@
+{- | Module: LambdaBuffers.Utils.Logger
+
+Utilities for logging INFO, and ERROR messages.
+
+This follows the GNU error message format:
+<https://www.gnu.org/prep/standards/html_node/Errors.html>
+-}
+module LambdaBuffers.Utils.Logger (logInfo, logError, logErrorWithSourceSpan) where
+
+import System.IO qualified
+
+{- | Logs an INFO message of the format
+> <file>: info: <msg>
+if @<file>@ is non empty, otherwise, it prints
+> info: <msg>
+-}
+logInfo :: FilePath -> String -> IO ()
+logInfo "" msg = System.IO.hPutStrLn System.IO.stderr $ "info: " <> msg
```

Comment:
shouldn't this be `stdout`?

Reply:
Honestly, I don't think so -- most of the time `stdout` is reserved for the actual output of the program, and `stderr` is for printing diagnostic and error messages. The actual output of compilers is usually done on just files, and everything else is then just diagnostic / errors which hence should go on stderr.

In the case of LB, `info` messages generally seem like "diagnostic" and hence **should** go to `stderr`.

But I'll change it to `stdout` anyways :P.

References
- https://linux.die.net/man/3/stderr
  where we quote
  > Under normal circumstances every UNIX program has three streams opened for it when it starts up, one for input, one for output, and **one for printing diagnostic or error messages**.